portfolio/src/main/webapp/css/custom.css

```diff
@@ -0,0 +1,97 @@
+@charset "UTF-8";
+/*!
+* Start Bootstrap - Agency v6.0.2 (https://startbootstrap.com/template-overviews/agency)
+* Copyright 2013-2020 Start Bootstrap
+* Licensed under MIT (https://github.com/StartBootstrap/startbootstrap-agency/blob/master/LICENSE)
+*/
+/*!
+ * Bootstrap v4.5.0 (https://getbootstrap.com/)
+ * Copyright 2011-2020 The Bootstrap Authors
+ * Copyright 2011-2020 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
+/* button  - #006600
+Button hover -  #004d00
+Button outline - #003300
+Text color -  #006600
+Gray - #6c757d
+*/
+header.masthead {
+  color: #fff;
+  background-image: url('../assets/img/header.jpg');
+  background-size: cover;
+}
+
+.btn-primary {
+  color: #fff;
+  background-color: #006600;
+  border-color: #006600;
+}
+.btn-primary:hover {
+  color: #fff;
+  background-color: #004d00;
+  border-color: #004d00;
+}
+.btn-primary:focus,
+.btn-primary.focus {
+  color: #fff;
+  background-color: #006600;
+  border-color: #004d00;
+  box-shadow: 0 0 0 0.2rem rgba(254, 216, 84, 0.5);
+}
+.btn-primary.disabled,
+.btn-primary:disabled {
+  color: #fff;
+  background-color: #006600;
+  border-color: #006600;
+}
+.btn-primary:not(:disabled):not(.disabled):active,
+.btn-primary:not(:disabled):not(.disabled).active,
+.show > .btn-primary.dropdown-toggle {
+  color: #fff;
+  background-color: #004d00;
+  border-color: #006600;
+}
```

Comment:
Add space between each class for consistency

Reply:
added